main/views/billing.py

```diff
@@ -499,10 +506,11 @@ def billing_stripe_webhook(request):
     See: https://stripe.com/docs/webhooks
     """
 
-    stripe.api_key = settings.STRIPE_API_KEY
+    stripe = StripeClient(api_key=settings.STRIPE_API_KEY)
     data = json.loads(request.body)
 
     try:
+        # TODO: I actually don't know how to do this on the new api
         event = stripe.Event.construct_from(data, stripe.api_key)
```

Comment:
This is the only part that is not straightforward to migrate. I need to read the docs to see how to do it.